modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -52,6 +52,40 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
 
 
 
+== Bootstrap End-of-Life {centos} 6 Clients
+
+{centos} 6 is now at end-of-life, and the ISO images provided in the client repository for this operating system are out of date.
```

Comment:
Not just the ISOs. Most likely the cloud images (qcow2) provided by CentOS as well. No idea about the cloud images at AWS or other Cloud providers, but my guess is that they are affected as well.